instructions.texi

```diff
@@ -396,13 +396,13 @@ control-flow constructs
 @item Stack effect:
 @math{-1+0}
 @item Example:
-With bytecode optimization, @code{(defun goto-if-not-nil-eg(n) (if (or (n) (n)) 1320))} generates
```

Comment:
Note, we probably have to go back to the other goto examples that assume no optimization an note they assume that explicitly. Bytecode optimization is on by default.

Reply:
So you're saying it should be on when nothing is specified? I agree, though there are a few examples that need converting for that...

Reply:
Well, only if it matters. In the goto examples it matters.